main screens (no backend)

nick-bash opened the pull request:
our 5 screens, with elements all in place
no API, mongo, or auth calls built yet

nick-bash committed:
main screens (no backend)

our 5 screens, with elements all in place
no API, mongo, or auth calls built yet

devlyn37 reviewed (approved):
Probably better to just use yarn here, also probably want to fix those deployment problems

eduairet committed:
Landing styling

nick-bash committed:
csv file integration

- employer/manageStreams now accepts csv files, turns them into array of JSON
- ready to connect with backend

nick-bash committed:
Fix build with eslint ignore

i hate eslint, lol

nick-bash committed:
retry to fix build errors

issue seems to be capitalizing next libraries accidentally
